solutions/standard/variables.tf

```diff
@@ -68,26 +68,20 @@ variable "secret_manager_tags" {
 
 variable "skip_kms_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all Secrets Manager instances in the resource group to read the encryption key from the KMS instance. If set to false, pass in a value for the KMS instance in the existing_kms_instance_guid variable."
-  default     = false
+  description = "A flag to create an IAM authorization that grants Secrets Manager access to a Key Protect resource(true) or not(false). Set to false if a preexisting authorization can be used."
```

Comment:
description should match whatever format we go with for `skip_en_iam_authorization_policy`

Reply:
So if you go with my suggestion for `skip_en_iam_authorization_policy`,

Whether to create an IAM authorization policy to permit Secrets Manager instances to read the encryption key from the KMS instance. Set to `false` to create the policy, and specify the KMS instance in the `existing_kms_instance_guid` variable. Set to `true` if a policy with the correct access exists.